_develop/connect/get-started.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: learn
+permalink: /:collection/:path.html
+---
+# Get Started
+{:.no_toc}
+
+Blockstack Connect is a Javascript library for integrating your application with Stacks v2. With Connect, you get some big benefits:
+
+<!-- -  -->
+
+* TOC
+{:toc}
+
+## Installation
+
+With yarn:
+
+```bash
+yarn add @blockstack/connect
+```
+
+With npm:
+
+```bash
+npm install --save @blockstack/connect
+```
+
+## Usage
+
```

Comment:
Maybe an overview of the 4 sub sections would help. Not sure if starting with AuthOptions without more context will be understandable